bin/run

```diff
@@ -34,9 +34,14 @@ async function main() {
   } catch(err) {
     await require('@oclif/errors/handle')(err);
 
-    console.log(err.stack);
-    console.log(err.code);
-    console.log(err.message);
+    let errInfo = ['stack', 'code', 'message'];
```

Comment:
This is unnecessarily complicated. Good spot on logging `undefined`s but why not just do `console.log(err.stack || '')` instead?

Reply:
That was my initial solution, I thought that it looks ugly, but It seems like I've made it worse 😅